docs/dev/apis/lethallib.md

```diff
@@ -0,0 +1,37 @@
+---
+prev: false
+next: false
+description: An list of entries for LethalLib modding topics and the basis for all LethalLib guides on this wiki.
+---
+# LethalLib mod creation basics
+This page will go over all the basics needed to start making mods using LethalLib. At the bottom of this page you can branch out to whatever specific part of the game you'd like to mod from there.
+
+## LethalLib requirements
+To begin with LethalLib development, you will need the following mods:
+
+- [BepinExPack by BepInEx](https://thunderstore.io/c/lethal-company/p/BepInEx/BepInExPack/)
+
+- [HookGenPatcher by Evaisa](https://thunderstore.io/c/lethal-company/p/Evaisa/HookGenPatcher/)
+
+- [LethalLib by Evaisa](https://thunderstore.io/c/lethal-company/p/Evaisa/LethalLib/)
+
+These can be installed with a mod manager such as r2modman or Thunderstore manager, or installed manually. A mod manager will let you automatically update LethalLib, but a manual install makes it easier to do LAN multiplayer testing. The guides below will assume a manual install for simplicity.
+
+## Development environment
+You will need to start by creating a plugin (see [Starting a Mod](/dev/starting-a-mod)) and adding the LethalLib .dll as a dependency. You do this in the same way you add the game assembly (see Starting a Mod again). If you manually installed LethalLib, you'd find it somewhere in `...\steamapps\common\Lethal Company\BepInEx\plugins`. If you installed via a mod manager, you can get to the mod profile folder where it will be located in a similar path like so:
+
+![Image of r2modman, highlighting the 'Browse profile folder' option](/images/lethallib/ProfileFolderModman.png)
+
+You'll also want to add the LethalLib mod ID as a BepinEx dependency, which can be done by adding the following line after `[BepinPlugin(...)]`:
+`[BepInDependency(LethalLib.Plugin.ModGUID)]`
```

Comment:
I would switch it to an actual code block with the added line being highlighted with either `// [!code highlight]`, `// [!code focus]`, or the best option `// [!code ++]`.

Ex:
`````md
text...

```cs
[Bepin...
added line // [!code ++]
private static...
```
`````